db/data/20210721181311_create_test_experiment.rb

```diff
@@ -0,0 +1,31 @@
+class CreateTestExperiment < ActiveRecord::Migration[5.2]
+  def up
+    if CountryConfig.current_country?("India")
+      experiment = Experimentation::Experiment.create!(name: "production test", experiment_type: "current_patients", state: "new")
```

Comment:
It may be worth pulling this out to a service object and calling it from this data migration -- that way you can easily run it locally and also add some tests to verify things.

Reply:
That's probably a better idea, definitely easier to work with that.